src/components/Settings/Profile/index.tsx

```diff
@@ -32,8 +32,8 @@ const ProfileSettings: NextPage = () => {
     variables: { request: { profileId: currentProfile?.id } },
     skip: !currentProfile?.id,
     onCompleted: (data) => {
-      // @ts-ignore
-      setSettingsType(data?.profile?.picture?.uri ? 'NFT' : 'AVATAR');
+      const picture = data?.profile?.picture;
+      setSettingsType(picture && picture.hasOwnProperty('uri') ? 'NFT' : 'AVATAR');
```

Comment:
I think we can replace `x && x.y` with optional chaining `x?.y` in similar other places too
```suggestion
      setSettingsType(picture?.hasOwnProperty('uri') ? 'NFT' : 'AVATAR');
```

Reply:
I have submitted the suggestion. Thanks!

Regarding the second comment, I think the behaviour is the same. It would set 'NFT' as settings type only if `picture.uri` exists. If it is `null | undefined` it will set it to `AVATAR`. I have made a little [TS Playground](https://www.typescriptlang.org/play?#code/MYewdgzgLgBADgJxAMwJYBsCmB1VUAWAcgGIAqMAvDAN4BQMDMArgqgFwwDkJpntAvgG5atUJFiIUGHHnwgmUHpRr1GIVgHNUYAIboOdRkeYJ9XAIIww6gLZ6YAVQBKAGT5H+A4aPAQQWADp0EA0ACkk0LFwCHgB+APwdCAB5AHcwAAUkOEwEKABPUM4WVE4AShhYrh5OGA5OcwA1c1JzJ3LhMT9A4LCI6Wi5BTiEpLTM7NyCopLyyuqyWvqmlraOkS7-TCCQ0KYwABNMNDBMA-nuRbqLZtb2ss7fLZ2wsCZ0dAua64bbtYfaEA) to check it and clarify myself too 😅

Reply:
> If it is null | undefined it will set it to AVATAR.

No, the `hasOwnProperty()` method will return true if a property exists and false if it doesn't. what if the 
```js
const profile = {
uri: null
}
```
this will set 'NFT' as type because `uri` property exists on profile, `hasOwnProperty()` doesn't care about uri's value . I've added a [TS playground](https://www.typescriptlang.org/play?#code/MYewdgzgLgBADgJxAMwJYBsCmB1VUAWAcgGIAqMAvDAN4BQMDMArgqgFwxhPrq0C+Ablq1QkEFgB06EAHMAFIhQYceImQD8E-AEMIAeQDuYAApI4mBFACecgOQtUtgJQx1MWyVK2YHWwEEANT9SPwAlZyFaIA)

## Before this PR
```js
setSettingsType(data?.profile?.picture?.uri ? 'NFT' : 'AVATAR');
```

## In this PR

```js
setSettingsType(picture?.hasOwnProperty('uri') ? 'NFT' : 'AVATAR');
```